doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -477,41 +479,9 @@ Supported settings
     A positive number greater than 1.0 controlling how fast memory increases
```

Comment:
Since some of these reference others, I probably would reorder them:  numberOfRetries first (I don't know if I would mention the default here if memoryMultiplier set or not), memoryMultiplier next (with that default for numberOfRetries here), memoryLimit, then requestMemoryMax.  [or alphabetize this entire section, but that's a lot more work]  I would also move the "It has no effect if memoryMultiplier is not set" to be the first (or second) sentence.  Not sure but maybe a pointer in the memory ones to the memory sections?

Reply:
I'll reorder as requested.